src/schemaExporter.ts

```diff
@@ -46,6 +46,12 @@ export class SchemaExporter implements IExporter {
 	private createAndSaveSnapshot = async () => {
 		const svc = this._getSchemaService();
 		let snapshot = await svc.snapshot();
+		snapshot.fields
+			.filter( (field: any) => field.schema?.default_value=="nextval('"+field.schema?.table+"_"+field.schema?.name+"_seq'::regclass)" )
```

Comment:
Firstly: You are basically looping in the worse case 2N just do a single forEach.
Secondly: This is very specific to postgres and as such should be extracted into its own file and only be executed if and when running on postgres database.